aten/src/ATen/CMakeLists.txt

```diff
@@ -329,7 +329,7 @@ IF(NOT TARGET cpuinfo)
   SET(CPUINFO_BUILD_UNIT_TESTS OFF CACHE BOOL "")
   SET(CPUINFO_BUILD_MOCK_TESTS OFF CACHE BOOL "")
   SET(CPUINFO_BUILD_BENCHMARKS OFF CACHE BOOL "")
```

Comment:
I suggest to copy the code in Caffe2 (with modification for `CPUINFO_SOURCE_DIR` w.r.t. ATen path), see: https://github.com/pytorch/pytorch/blob/master/cmake/Dependencies.cmake#L95-L118
This will support some nice options:
- Overriding `cpuinfo` location for development by specifying `-DCPUINFO_SOURCE_DIR=/path/to/your/cpuinfo/checkout`.
- Avoiding double-import of `cpuinfo` when `cpuinfo` is already imported through another dependency.
- Linking `cpuinfo` statically even when other libs are linked dynamically. This prevents issues with mulitple cpuinfo versions overwriting each other, e.g. in conda installation.